### PR TITLE
{App Service} Add to docs: function app appsettings command supports json file input

### PR DESCRIPTION
**Pull request by amasover**
**Related command**

az functionapp config appsettings set

**Description**

I noticed that both

```
az functionapp config appsettings set
```
and
```
az webapp config appsettings set
```
support syntax where you can pass in a json file containing appsettings with `@file.json`.

However, previously only webapps had this syntax documented. This PR is to add the syntax documentation for the functionapps command.

**Testing Guide**

```
az functionapp config appsettings set -g MyResourceGroup -n MyUniqueApp --settings mySetting=value @moreSettings.json
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
App Service

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py [hunk 1]
```diff
@@ -205,6 +205,14 @@
...
     text: |
         az functionapp config appsettings set --name MyFunctionApp --resource-group MyResourceGroup --settings "AzureWebJobsStorage=$storageConnectionString"
+  - name: Set using both key-value pair and a json file with more settings.
+    text: >
+        az functionapp config appsettings set -g MyResourceGroup -n MyUniqueApp --settings mySetting=value @moreSettings.json
+parameters:
+  - name: --settings
+    short-summary: Space-separated appsettings in KEY=VALUE format. Use @{file} to load from a file.
+  - name: --slot-settings
+    short-summary: Space-separated appsettings in KEY=VALUE format. Use @{file} to load from a file. Given setting are added to the configuration and marked as Deployment slot setting by default.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/appservice/_help.py:215]: The `parameters:` part should be written above `examples:` part

Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/appservice/_help.py:215]: The `parameters:` part should be written above `examples:`.

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/appservice/_help.py:215]: Please help also revise this code: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/_help.py#L1103-L1107

**Comment by amasover** [comment]
@microsoft-github-policy-service agree

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py (hunk 1 above)
Comment (amasover) [reply src/azure-cli/azure/cli/command_modules/appservice/_help.py]: Apologies for the delay. I've moved the `parameters:` part above the `examples:` part. This should be ready for review again.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**